test/user.test.js

```diff
@@ -678,6 +760,22 @@ describe('User', function() {
           done();
         });
     });
+
+    it('allows login with password too long but created in old LB version',
+    function(done) {
+      var bcrypt = require('bcryptjs');
+      var longPassword = new Array(80).join('x');
+      var oldHash = bcrypt.hashSync(longPassword, bcrypt.genSaltSync(1));
+
+      User.create({ email: 'b@c.com', password: oldHash }, function(err) {
+        if (err) return done(err);
+        User.login({ email: 'b@c.com', password: longPassword }, function(err) {
+          if (err) return done(err);
+          // we are logged in, the test passed
```

Comment:
Please assert the user is logged in instead of adding a comment (ie. verify the token, etc).